queryx.go

```diff
@@ -136,7 +144,7 @@ func (q *Queryx) BindStructMap(arg0 interface{}, arg1 map[string]interface{}) *Q
 	return q
 }
 
-func bindStructArgs(names []string, arg0 interface{}, arg1 map[string]interface{}, m *reflectx.Mapper) ([]interface{}, error) {
+func bindStructArgs(names []string, arg0 interface{}, arg1 map[string]interface{}, m *reflectx.Mapper, tr bindTransformer) ([]interface{}, error) {
```

Comment:
It already has 2 Query fields passed in let's make it a method.

Reply:
Done